CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
```

Comment:
This wasn't public API to begin with so I don't think it needs a changelog personally.

Reply:
Otherwise this LGTM

Reply:
Removed